### DISCUSSION TITLE
Custom memoize per input selector?

### DISCUSSION THREAD

**Discussion by denchen** [reactjs/reselect#496] [Q&A]
Using `reselect` with Redux, is there a way to have a custom memoize for only the props I pass into the selector, and not the ones that come from Redux? As an example:

```typescript
const selectCompletedTodosCount = createSelector(
  state => state.todos, // <-- I want this to use the default memoize
  (_, completed) => completed, // <-- I want a custom memoize for just this arg
  (todos, completed) =>
    todos.filter(todo => todo.completed === completed).length
)
```

I'm reading through the documentation on custom memoization and `createSelectorCreator()`, and I didn't see anything about the memoize function being able to tell which input selector it's currently checking.

**Comment by gabrielpetersson** [comment]
Yeah here, you can pass `defaultMemoize` for the first option and a custom memoizer for the second
[createSelectorCreator](https://github.com/reduxjs/reselect#createselectorcreatormemoize-memoizeoptions)
